ilmsdump.ui

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
(packaging)

move inside `ilmsdump/gui/`

setup.py would need package_data=XXX to include this file